crates/types/build.rs

```diff
@@ -0,0 +1,23 @@
+extern crate rjacraft_data_generator;
+
+use rjacraft_data_generator::*;
+
+fn main() {
+    use std::{fs::OpenOptions, path::Path};
+
+    // Only re-run if the Minecraft JSON data files were modified.
+    //println!("cargo:rerun-if-changed=../../mc-data/blocks.json");
+
+    let out_dir = std::env::var_os("OUT_DIR").expect("environment variable OUT_DIR set");
+    let mut dest_file = OpenOptions::new()
+        .write(true)
+        .create(true)
+        .truncate(true)
+        .open(Path::new(&out_dir).join("blocks.rs"))
+        .expect("create destination file");
+
+    let src_path = Path::new("../../mc-data/blocks.json");
```

Comment:
Deal with the source of data–should we download it ourselves, or have users bring a `blocks.json` of their own?

Reply:
Let's shed some light on this. What have we settled on? @rjacraft/developer

Reply:
Don't download anything and just make the path configurable